server/server.go

```diff
@@ -653,20 +661,20 @@ func (server *Server) biomes() map[string]interface{} {
 // loadResources loads resource packs from path of specifed directory.
 func (server *Server) loadResources(p string, log internal.Logger) {
 	if _, err := os.Stat(p); os.IsNotExist(err) {
-		_ = os.Mkdir(p, 0777)
+		_ = os.Mkdir(p, os.ModePerm)
```

Comment:
This makes it less obvious which permissions are used, I'd just leave this as 0777.

Reply:
Reverted.